app/components/application/site-search/group.js

```diff
@@ -1,26 +1,10 @@
 import Component from 'ember-component';
-import get from 'ember-metal/get';
-import service from 'ember-service/inject';
-import { typeOf } from 'ember-utils';
 import { invokeAction } from 'ember-invoke-action';
-import { modelType } from 'client/helpers/model-type';
 
 export default Component.extend({
-  router: service('-routing'),
-
   actions: {
-    transitionTo(item) {
+    close() {
```

Comment:
✂️ as we aren't doing any additional logic with this action, we can pass it directly to the child component: `close=close`